dftimewolf/lib/collectors/gce_disk_export.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Export disk image from a GCP project to Google Cloud Storage."""
+
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+from libcloudforensics import gcp
+
+from dftimewolf.lib import module
+from dftimewolf.lib.modules import manager as modules_manager
+
+
+class GoogleCloudDiskExport(module.BaseModule):
```

Comment:
Also with a name like this, it should probably go in `/exporters/`

Reply:
Yes, that makes sense, will move it to exporters.